src/ts/components/seekbar.ts

```diff
@@ -207,46 +207,62 @@ export class SeekBar extends Component<SeekBarConfig> {
     let isUserSeeking = false;
     let isPlayerSeeking = false;
 
+    const getPlaybackPositionPercentage = (): number => {
+      if (player.isLive()) {
+        return 100 - (100 / player.getMaxTimeShift() * player.getTimeShift());
+      }
+
+      return 100 / player.getDuration() * this.getRelativeCurrentTime();
+    };
+
+    const updateBufferLevel = (playbackPositionPercentage: number): void => {
+      if (player.isLive()) {
+        // Always show full buffer for live streams
+        this.setBufferPosition(100);
+        return;
+      }
+
+      const playerDuration = player.getDuration();
+
+      const videoBufferLength = player.getVideoBufferLength();
+      const audioBufferLength = player.getAudioBufferLength();
+      // Calculate the buffer length which is the smaller length of the audio and video buffers. If one of these
+      // buffers is not available, we set it's value to MAX_VALUE to make sure that the other real value is taken
+      // as the buffer length.
+      let bufferLength = Math.min(
+          videoBufferLength != null ? videoBufferLength : Number.MAX_VALUE,
+          audioBufferLength != null ? audioBufferLength : Number.MAX_VALUE);
+      // If both buffer lengths are missing, we set the buffer length to zero
```

Comment:
this could be a function as well

Reply:
Updated here https://github.com/bitmovin/bitmovin-player-ui/pull/414/commits/d0781dd012a4a1c057f7d1d476bd8d9a0164b893

Reply:
Sry i meant this whole block, could be a function `getMinBufferLevel`

Reply:
https://github.com/bitmovin/bitmovin-player-ui/pull/414/commits/49b96bf065c026f15e48fb846086e98b2ff6d2a7